task2/task2.php

```diff
@@ -0,0 +1,7 @@
+<?php
+function multipleAddition(int $x): int
+{
+    $number = array_sum(array_map('intval', str_split($x)));
+    return count($number) !== 1 ? multipleAddition($number) : $number;
```

Comment:
PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, int given. See https://www.php.net/count